cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1118,12 +1118,33 @@ function WaitFor_KubeletAndKubeProxyReady {
 }
 
 # Runs 'kubectl get nodes'.
-# TODO(pjh): run more verification commands.
+# Runs additional verification commands to ensure node succesfully joined cluster
+# and that it connects to the API Server.
 function Verify-WorkerServices {
-  Log-Output ("kubectl get nodes:`n" +
-              $(& "${env:NODE_DIR}\kubectl.exe" get nodes | Out-String))
+  $timeout = 12
```

Comment:
nit: keep a variable with sleep period as well, like $retryDelayInSeconds. to be easier to figure the total retry time.

Reply:
will do, thank you